docs/cita/economics/charge.md

```diff
@@ -26,7 +26,6 @@ $ bin/cita create --init_token 0x2540BE400 --contract_arguments SysConfig.econom
 CITA 提供灵活的分发方式，以适配多样的业务需求:
 
 * 超级管理员可以通过发交易的方式进行分配；
-* 也可以使用我们提供的[水龙头工具](https://github.com/cryptape/testnet-faucet-mri)，让用户自己领取代币；
```

Comment:
But we can use faucet now!

Reply:
I think it should only work in the quota model.

Reply:
So, close this PR?